README.md

```diff
@@ -1 +1,232 @@
-# JavaScript-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Microverse README Template</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 JS Capstone Module 2](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [JS Capstone Module 2] <a name="about-project"></a>
+
+> Its a daily rountine list for users to write down their task.
+
+**[JS Capstone]** is a project for microvesre.
+
```

Comment:
- [ ] Please let's improve the project description text 👎🏻 

![image](https://user-images.githubusercontent.com/79058364/235335425-551882b7-6a59-4d6a-9448-e79d0a5bfef4.png)

Reply:
well resolved! ✔️